test/Main.hs

```diff
@@ -146,6 +146,7 @@ conversionTo = do
   quickCheckRoundtrip @Datetime2 "Datetime2" "datetime2"
   quickCheckRoundtrip @Smalldatetime "Smalldatetime" "smalldatetime"
   quickCheckRoundtrip @TestDateTime "TestDateTime" "datetime"
+  quickCheckRoundtrip @Datetimeoffset "Datetimeoffset" "datetimeoffset"
```

Comment:
A roundtrip test makes me much more confident in the correctness. :+1: :slightly_smiling_face: